site/css/style.css

```diff
@@ -336,6 +336,12 @@ fieldset {
     border: none;
 }
 
+fieldset > div {
```

Comment:
Instead of `fieldset > div`, should this be `.user-input-fields fieldset > div`,
so that you only target fieldsets that request it?